app/scripts/modules/core/src/cluster/filter/FilterSearch.tsx

```diff
@@ -20,6 +20,11 @@ export const FilterSearch = ({ helpKey, onBlur, onSearchChange, value }: IFilter
           value={value}
           onBlur={onBlur}
           onChange={onSearchChange}
+          onKeyUp={(event: React.KeyboardEvent<HTMLInputElement>) => {
+            if (event.which == 13) {
```

Comment:
Is `event.which` sometimes a `string`? Just wondering if there is a reason for double equals vs triple.

Reply:
My bad, didn't notice I had used only double equals